src/components/MatchScore/MatchScore.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import styles from './matchScore.module.scss';
+import { classNames } from '../../shared/utilities';
+import { MatchScoreProps } from './MatchScore.types';
+
+export const MatchScore: FC<MatchScoreProps> = ({
+    className,
+    score = 0,
+    total = 5,
+    hideLabel = false,
+}) => {
+    const absTotal = Math.abs(total);
+    const matchScoreClasses = classNames(className, styles.matchScoreContainer);
+    return (
+        <div
+            className={matchScoreClasses}
+            role="progressbar"
+            aria-valuenow={score}
+            aria-valuemax={total}
```

Comment:
Yeah i'm not sure progressbar is the correct role here. @dkilgore-eightfold can chime in on this